app/models/content_configuration.rb

```diff
@@ -23,25 +23,29 @@ class ContentConfiguration < Spree::Preferences::Configuration
   # Producer sign-up page
   # All the following defaults using I18n don't work.
   # https://github.com/openfoodfoundation/openfoodnetwork/issues/3816
-  preference :producer_signup_pricing_table_html, :text,
-             default: I18n.t(:content_configuration_pricing_table)
-  preference :producer_signup_case_studies_html, :text,
-             default: I18n.t(:content_configuration_case_studies)
-  preference :producer_signup_detail_html, :text, default: I18n.t(:content_configuration_detail)
+  # default values for these fields are commented out below
+  preference :producer_signup_pricing_table_html, :text
+  # default: I18n.t(:content_configuration_pricing_table)
+  preference :producer_signup_case_studies_html, :text
+  # default: I18n.t(:content_configuration_case_studies)
+  preference :producer_signup_detail_html, :text
+  # default: I18n.t(:content_configuration_detail)
```

Comment:
This needs to be resolved before we merge the PR. Otherwise the default values will be lost in production.

Reply:
Done now :+1:  Happy to see that the current change in base spec helper does not break the build, if we keep this setting (as opposed to the initial proposal, in this PR).